contracts/src/v0.8/Verifier.sol

```diff
@@ -394,7 +425,7 @@ contract Verifier is IVerifier, ConfirmedOwner, TypeAndVersionInterface {
 
     feedVerifierState.s_verificationDataConfigs[configDigest].f = f;
     feedVerifierState.s_verificationDataConfigs[configDigest].isActive = true;
-    for (uint8 i; i < signers.length; i++) {
+    for (uint8 i; i < signers.length; ++i) {
```

Comment:
What is the difference? Is this one more efficient?

Reply:
By the way is it still inefficient to call `signers.length` every time in the for loop or did solidity add an optimisation for that?